website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -7,6 +7,10 @@ page_id: api-changelog
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+### 2022-06-17
+* Updated [Creating a Run Task](/cloud-docs/workspaces/settings/run-tasks#creating-a-run-task) section to include the new description information for the run task to be configured.
+* Update [Request Body](/cloud-docs/api-docs/run-tasks.mdx#request-body) to include the new description attribute.
```

Comment:
```suggestion
* Update [Run Tasks](/cloud-docs/api-docs/run-tasks.mdx) to include the new description attribute.
```

I feel we could be more inclusive of Run Tasks as a whole, since it isn't only one request or response that will have this new information.

I also want to check that the [Update a Run Task#Request Body](https://www.terraform.io/cloud-docs/api-docs/run-tasks#update-a-run-task) needs to be updated to. cc @glennsarti

Reply:
Good pickup!  yup that should be in there. I can't do suggestions on that but the relevant spots

https://github.com/hashicorp/terraform-docs-common/pull/48/files#diff-e50f26b0b4b230e942d74e03719d081761c2dceb7c18e0e5a7d9f016455cbe1cR313-R319

https://github.com/hashicorp/terraform-docs-common/pull/48/files#diff-e50f26b0b4b230e942d74e03719d081761c2dceb7c18e0e5a7d9f016455cbe1cR327-R332